ReSharper.FSharp/src/FSharp.Psi.Features/src/Util/PsiUtil.fs

```diff
@@ -141,19 +141,25 @@ let isSemicolon (node: ITreeNode) =
     getTokenType node == FSharpTokenType.SEMICOLON
 
 
-let rec skipTokensOfTypeAfter tokenType (node: ITreeNode) =
+let rec skipTokensOfTypesAfter tokenTypes (node: ITreeNode) =
     let nextSibling = node.NextSibling
-    if getTokenType nextSibling == tokenType then
-        skipTokensOfTypeAfter tokenType nextSibling
+    if tokenTypes |> List.contains (getTokenType nextSibling) then
```

Comment:
Please use token type sets instead going through list. Use either HashSet`1 or look at how token sets are defined in FSharpTokenNodeType.